### PR TITLE
create the order page

### LINKED ISSUE #6: create the header of order page
create the title, back arrow, and the discounts

### LINKED ISSUE #8: create the first order "coffee latte" in the main of order page
create the list of orders and their prices and style it.

### LINKED ISSUE #25: solve the scrolling in discounts
the boxes in the header are scrolled with the main page, solve the problem and make them scrolled alone.

### LINKED ISSUE #30: change the place of ammount photo in coffee latte

### LINKED ISSUE #31: create the second order in the main of order page

### LINKED ISSUE #32: create the third order in the main of order page

### LINKED ISSUE #33: create the total price

### LINKED ISSUE #34: create the footer of order page
create 2 boxes the first is for "+" and the second is for "checkout"

### PR DESCRIPTION
**Pull request by salwasiksik**
relates #6

### CODE REVIEW

**Review by MohamedSatary — APPROVED** [review]

File: order/order.html [hunk 1]
```diff
@@ -8,6 +8,68 @@
...
+            </div>
+            <div class="amount-pic1">
+              <img src="../img/Amount2.svg" class="amount1" alt="amount">
+            </div>
+          </section>
+    </div>
+    <hr>
+    <div class="totalnumber">
+        <p class="total">Total</p>
+        <p class="number">$ 29.96</p>
```
Comment (yaqootturman) [inline order/order.html]: total and the number should be in te same line

File: order/order.html [hunk 2]
```diff
@@ -8,6 +8,68 @@
...
+    </div>
```
Comment (yaqootturman) [inline order/order.html:73]: You miss last two elements in the page , check the Figma design

File: order/order.css [hunk 3]
```diff
@@ -0,0 +1,240 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Readex+Pro&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Readex+Pro:wght@200;400&display=swap");
+html {
```
Comment (yaqootturman) [inline order/order.css:3]: You have a large white space vertical scrolling

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: order/order.html (hunk 2 above)
Comment (salwasiksik) [reply order/order.html:73]: I added them in the issue #34

File: order/order.html (hunk 1 above)
Comment (salwasiksik) [reply order/order.html]: they are already in the same line in my pc

**Review by yaqootturman — APPROVED** [review]

**Review by MohamedSatary — APPROVED** [review]

**Review by bayan2002 — APPROVED** [review]

**Pull request merged**